site/index.md

```diff
@@ -9,9 +9,11 @@ layout: home
 #  a makerspace in Roanoke, VA.
 ---
 
+### MAKE Roanoke is launching a Kickstarter campaign to open a Makerspace in Roanoke, VA
+
```

Comment:
```suggestion
![](/assets/images/Kickstarter.png)
```

Reply:
We are no longer using that, and replacing this picture with the kickstarter party image:

https://discord.com/channels/1179210982258647100/1287907844695396385/1296583231554060348